src/modules/map/components/openlayers/OpenLayersHighlightedFeatures.vue

```diff
@@ -94,8 +95,12 @@ watch(nonEditableFeature, () => {
         nonEditableFeature.value.filter((feature) =>
             ['Point', 'MultiPoint'].includes(feature.geometry?.type)
         ).length === nonEditableFeature.value.length
-    if (isFloatingTooltip.value && !containsOnlyPoints) {
-        toggleFloatingTooltip()
+    if (!isBottomPanelFeatureInfo.value && !containsOnlyPoints) {
+        // we need to also go to 'BOTTOMPANEL' if we are on 'NONE', here
+        store.dispatch('setFeatureInfoPosition', {
+            featureInfo: FeatureInfoPositions.BOTTOMPANEL,
+            dispatcher: dispatcher,
+        })
```

Comment:
Would this not be an issue (a contradiction) when the featureInfo parameter is not present or None ?